src/test/java/com/eleks/academy/whoami/repository/GameInMemoryRepositoryTest.java

```diff
@@ -0,0 +1,49 @@
+package com.eleks.academy.whoami.repository;
+
+import com.eleks.academy.whoami.core.SynchronousGame;
+import com.eleks.academy.whoami.core.impl.PersistentGame;
+import com.eleks.academy.whoami.model.request.NewGameRequest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+public class GameInMemoryRepositoryTest {
+
+	@Mock
+	private final Map<String, SynchronousGame> games = new ConcurrentHashMap<>();
+
+	private final NewGameRequest gameRequest = new NewGameRequest();
+
+	@BeforeEach
+	public void setMockMvc() {
+		gameRequest.setMaxPlayers(5);
+	}
+
+	@Test
+	void save() {
+		final String player = "player";
```

Comment:
suggest you look at the test one more time in the title of the test is written `GameInMemoryRepositoryTest`, but I can't find any usage or test of `GameInMemoryRepository.class` and his methods

Reply:
Resolved
But I am not clearly sure if it is correct